python-threatexchange/threatexchange/exchanges/impl/techagainstterrorism_api.py

```diff
@@ -0,0 +1,129 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+
+"""SignalExchangeAPI implementation for Tech Against Terrorism Hash List API"""
+
+
+import typing as t
+from dataclasses import dataclass
+
+from threatexchange.exchanges.clients.techagainstterrorism import api
+from threatexchange.exchanges import fetch_state as state
+from threatexchange.exchanges import signal_exchange_api
+from threatexchange.exchanges import auth
+from threatexchange.exchanges.collab_config import (
+    CollaborationConfigWithDefaults,
+)
+from threatexchange.signal_type.signal_base import SignalType
+from threatexchange.signal_type.pdq.signal import PdqSignal
+from threatexchange.signal_type.md5 import VideoMD5Signal
+
+_API_NAME: str = "tat"
```

Comment:
blocking: It's more confusing to have this outside than inlined. I think in the place where you are copying it from, it is being reused somewhere.